gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -70,23 +70,23 @@ func NewVMKeeper(
 }
 
 func (vm *VMKeeper) Initialize(ms store.MultiStore) {
-	if vm.gnoStore != nil {
```

Comment:
please revert, gnoStore makes sense, as the SDK has its own "store", so this helps distinguish between the gno store and the underlying store.

Reply:
sure.